### ISSUE TITLE
Difference in kmeans initialization produces slightly different results

### ISSUE THREAD

**Issue by rbavery** [agroimpacts/cloud-free-planet#11]
Zhu used the following method to initialize kmeans, where `data` is the sample of 10000 values from the HOT time series.

```
;get initial center
center=fltarr(n_variable,k_class)
for i_v=0,n_variable-1, 1 do begin
    meanv=mean(data[i_v,*])
    sdv=stddev(data[i_v,*])
    lower=max([meanv-2.0*sdv,min(data[i_v,*])])
    upper=min([meanv+2.0*sdv,max(data[i_v,*])])
    interval=(upper-lower)/(k_class)
    center[i_v,*]=lower+(indgen(k_class)+0.5)*interval
endfor
```

Not totally sure what's going on here, but I think the 3 centers are being initialized relative to the mean, assuming a normal distribution.

Whereas we use a method similar to random init, kmeans++, https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html

The mask results end up looking very similar for my test time series so this isn't a problem yet, but noting it since it the main source of difference between the idl and python cloud mask results.
